oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -1,37 +1,37 @@
 /**
  * Oshi (https://github.com/oshi/oshi)
- *
+ * <p>

```

Comment:
I'll merge like this, but unless we change the license header before the next release (snapshot) they'll get reverted.  I guess I can add these paragraph breaks to the license header though.